scripts/tests/chiptest/__init__.py

```diff
@@ -116,19 +64,110 @@ def _GetManualTests() -> Set[ManualTest]:
     #
     # This is on purpose for now to make it harder to orphan files, however
     # we can reconsider as things evolve.
-    manualtests.add(ManualTest(yaml="Config_Example.yaml", reason="Example"))
-    manualtests.add(ManualTest(yaml="Config_Variables_Example.yaml", reason="Example"))
-    manualtests.add(ManualTest(yaml="PICS_Example.yaml", reason="Example"))
-    manualtests.add(ManualTest(yaml="Response_Example.yaml", reason="Example"))
-    manualtests.add(ManualTest(yaml="Test_Example.yaml", reason="Example"))
-
-    # Flagged as manual from: src/app/tests/suites/manualTests.json
-    for item in _LoadManualTestsJson(os.path.join(_YAML_TEST_SUITE_PATH, "manualTests.json")):
-        manualtests.add(item)
+    manualtests.add("Config_Example.yaml")
```

Comment:
Do we want to add example to it's own category. I know something like PICS_Example.yaml will never work, so I wouldn't want that to break someones manual flow in the future

Reply:
True, I will mark these as completely excluded then.